### PR TITLE
Add Page Visibility example

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR adds to the explainer the Page Visibility API to the list of APIs that interact well with Web NFC.

### CODE REVIEW

File: EXPLAINER.md [hunk 1]
```diff
@@ -149,6 +149,22 @@ document.querySelector("#abortButton").onclick = event => {
 };
 ```
 
+## Integration with Page Visibility
+
+Web NFC functionality is allowed only for the Document of the top-level browsing
```
Comment (kenchris) [inline EXPLAINER.md]: Since we are not linking to specs, I think it is fine to write document (lowercase)

File: EXPLAINER.md [hunk 2]
```diff
@@ -149,6 +149,22 @@ document.querySelector("#abortButton").onclick = event => {
...
+context, which must be visible. Thanks to the Page Visibility API, web
```
Comment (kenchris) [inline EXPLAINER.md]: Should probably link to spec

**Review by kenchris — APPROVED** [review]

File: EXPLAINER.md (hunk 1 above)
Comment (beaufortfrancois) [reply EXPLAINER.md]: Done

File: EXPLAINER.md (hunk 2 above)
Comment (beaufortfrancois) [reply EXPLAINER.md]: Good catch. Done.

**Review by zolkis — APPROVED** [review]
Thanks!

**Pull request merged**